presto-orc/src/main/java/com/facebook/presto/orc/metadata/CompressionKind.java

```diff
@@ -15,5 +15,5 @@
 
 public enum CompressionKind
 {
-    UNCOMPRESSED, ZLIB, SNAPPY
+    UNCOMPRESSED, ZLIB, SNAPPY, ENCRYPTED
```

Comment:
@martint, Presto defined its own CompressionKind. Do we have to have exact match between this one and the Apache one? For example, 'UNCOMPRESSED' is only roughly mapped to 'NONE' there. Thanks!